docs/politics.md

```diff
@@ -26,41 +26,41 @@ In politics, it can be useful to know which particular ideology someone subscrib
+A lot of political debate actually just boils down to having some differing, fundamental position. Instead of two people arguing at a surface level about an issue, it is sometimes more useful to dive down to try and figure out what the other person truly believes at a fundamental level. A great example of this is abortion — people will often debate back and forth about the legality of abortion while ignoring that they fundamentally disagree on whether or not a fetus should carry the same (or similar) moral consideration as an infant child.
 
 ## Libertarianism
 
 [Libertarianism](https://en.wikipedia.org/wiki/Libertarianism) is concerned with maximizing liberty and personal freedom. I viewed myself as a libertarian when I was a teenager and have read books by [Ayn Rand](https://en.wikipedia.org/wiki/Ayn_Rand).
 
-I am still pro liberty and pro freedom, but my political views have evolved. Libertarianism does not seem to do a very good job at solving some major problems, like social inequality and global warming. The latter, in particular, seems likely only to be solved by international cooperation at a global level.
+I am still pro-liberty and pro-freedom, but my political views have evolved. Libertarianism does not seem to do a very good job at solving some major problems, like social inequality and global warming. The latter, in particular, seems likely only to be solved by international cooperation at a global level.
 
 ## The Alt-Right
 
-I do not support the alt-right, nor any of the prescriptions they make for society. I do believe it is important, however, to acknowledge some of the conditions that lead people to becoming radicalized, e.g. feeling disconnected from society, having no sense of purpose, feeling economically left behind. I believe some on the alt-right are effective at identifying frustrations that one might have with our current economic or social system, but they offer no realistic solutions to any of these problems, and their explanations for said phenomenon are often mired in anecdote.
+I do not support the alt-right, nor any of the prescriptions they make for society. I do believe it is important, however, to acknowledge some of the conditions that cause people to become radicalized, e.g. feeling disconnected from society, having no sense of purpose, feeling economically left behind. I believe some on the alt-right are effective at identifying frustrations that one might have with our current economic or social system, but they offer no realistic solutions to any of these problems, and their explanations for a phenomenon are often mired in anecdote.
```

Comment:
```suggestion
I do not support the alt-right, nor any of the prescriptions they make for society. I do believe it is important, however, to acknowledge some of the conditions that cause people to become radicalized, e.g. feeling disconnected from society, having no sense of purpose, feeling economically left behind. I believe some on the alt-right are effective at identifying frustrations that one might have with our current economic or social system, but they offer no realistic solutions to any of these problems, and their explanations for said phenomenon are often mired in anecdote.
```

The word "said" is referring to the previously mentioned "frustrations". Saying "a phenomenon" is less specific and confusing as to what phenomenon is being referred to.

Reply:
I agree, but I would say that a rewording here would still make things clearer.